ctapipe/io/containers.py

```diff
@@ -75,6 +75,10 @@ class RawCameraContainer(Container):
     """
     Storage of raw data from a single telescope
     """
+    event_number = Item(-1, "telescope event number")
+    num_channels = Item(-1, "number of channels per pixel")
```

Comment:
> For now, it's fine to keep using the insturment container until this new API is ready, though, and it may even help us to not miss something useful.

Ok, I'll use it then